core/field_colour.js

```diff
@@ -160,25 +218,54 @@ Blockly.FieldColour.activateEyedropper_ = null;
 Blockly.FieldColour.EYEDROPPER_PATH = 'eyedropper.svg';
 
 /**
- * Set a custom colour grid for this field.
- * @param {Array.<string>} colours Array of colours for this block,
- *     or null to use default (Blockly.FieldColour.COLOURS).
- * @return {!Blockly.FieldColour} Returns itself (for method chaining).
+ * Create label and readout DOM elements, returning the readout
+ * @param {string} labelText - Text for the label
+ * @return {Array} The container node and the readout node.
+ * @private
  */
-Blockly.FieldColour.prototype.setColours = function(colours) {
-  this.colours_ = colours;
-  return this;
+Blockly.FieldColour.prototype.createLabelDom_ = function(labelText) {
+  var labelContainer = document.createElement('div');
+  labelContainer.setAttribute('class', 'scratchColorPickerLabel');
```

Comment:
most things in the api are spelled colour (with the 'u'). I don't think you want to start mixing them.